i18n/de_DE/messages/messages.json

```
@@ -526,6 +526,12 @@
   "update_address_page_title": "Adresse aktualisieren",
   "use_of_funds_page_title": "Mittelverwendung",
 
-  "language_selection": "Sprachauswahl öffnen"
-
+  "language_selection": "Sprachauswahl öffnen",
+
+  "donation_form_summary_title": "Zusammenfassung",
+  "donation_form_summary_contact_data": "Ihre Kontaktdaten",
+  "donation_form_summary_bank_data": "Bankdaten",
+  "donation_form_summary_iban": "IBAN",
+  "donation_form_summary_account_holder": "Kontoinhaber",
+  "donation_form_summary": "Sie spenden {interval} {formattedAmount} per {paymentType} als {personType}. Ihre Kontaktdaten:\n{address}\n{email}"
```

Please split this into payment and contact data (as you already did in line 532) and use the translation keys directly in the `<template>` section.

```suggestion
  "donation_form_summary_payment": "Sie spenden {interval} {formattedAmount} per {paymentType} als {personType}."
```

---

You won't be using the whole sentence as a language item. And have the sentence in two types. Becauseof the following requirement:
f the payment type is missing, the word "via" (en) and "per" (de) is not displayed in the summary box.

Maybe have two items:
One with payment type and one without payment type.